doc/nrf/protocols/matter/end_product/last_fabric_removal_delegate.rst

```diff
@@ -0,0 +1,85 @@
+.. _ug_matter_last_fabric_removal_delegate:
+
+Matter fabric removal
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Reaction to the last Matter fabric removal
+******************************************
+
+When a Matter device joins the Matter network it also joins the Matter Fabric created by an ecosystem.
```

Comment:
There isn't something called Matter network. Matter network is actually a fabric, so maybe we should try to change it to:
```suggestion
When a Matter device is commissioned by the Matter controller it joins the Matter Fabric created by the controller.
```